automatic/ant-renamer/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,19 @@
-﻿$packageName = '{{PackageName}}'
-$installerType = 'exe'
-$silentArgs = '/VERYSILENT'
-$url = '{{DownloadUrl}}'
+﻿$packageName  = 'alldup'
+$url          = 'http://update.antp.be/renamer/antrenamer2_install.exe'
+$checksum     = '866327c8c5e36e1e525da37ccdb40924e16ce5a5772ff67414b62b9d508cf2e5'
+$checksumType = 'sha256'
 
-Install-ChocolateyPackage $packageName $installerType $silentArgs $url
+$packageArgs  = @{
+  packageName    = $packageName
+  fileType       = 'exe'
+  softwareName   = 'Ant Renamer'

```

Comment:
I honestly don't know.
I just added it because of:
1. It is part of the template when running `choco new`
2. I thought perhaps it had something to do with Synchronizing with Porgams and features when using the Pro version.

I can remove it though if it truly has no meaning.